ext/strings.go

```diff
@@ -386,6 +437,17 @@ func lastIndexOfOffset(str, substr string, offset int64) (int64, error) {
 	return -1, nil
 }
 
+func lowerASCII(str string) string {
+	runes := []rune(str)
+	for i, r := range runes {
+		if r&unicode.MaxASCII == r {
```

Comment:
`if r <= unicode.MaxASCII` seems to be more idiomatic, but I'm okay with it as-is. Here and below.

Reply:
Done.